### PR TITLE
Update Puppet Providers Wechat 4u

### PR DESCRIPTION
**Pull request by Soumi7**
Updates References - Wechat4u  section in Puppet Providers

References issue #890

Screenshots :

image

image

image

### CODE REVIEW

**Review by shraddhavp — APPROVED** [review]

**Comment by shraddhavp** [comment]
LGTM

**Comment by shraddhavp** [comment]
please check the CLI once it has not turned green yet

**Comment by Soumi7** [comment]
Yes, I restarted the job, it has turned green now.

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

File: docusaurus/docs/puppet-providers/wechat4u.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ sidebar_label: WeChat4U
 [![NPM Version](https://badge.fury.io/js/wechaty-puppet-wechat4u.svg)](https://badge.fury.io/js/wechaty-puppet-wechat4u)
 [![npm (tag)](https://img.shields.io/npm/v/wechaty-puppet-wechat4u/next.svg)](https://www.npmjs.com/package/wechaty-puppet-wechat4u?activeTab=versions)
 
+Wechat4U is an excellent wechat bot framework that supports both Node.js & Browser, with rich features and an active community of experienced contributors. PuppetWechat4U will let wechaty users have the second way to use wechaty, not only by Browser Hook but also by Server API Call.
```
Comment (proudofsimin) [inline docusaurus/docs/puppet-providers/wechat4u.md]:
```suggestion
Wechat4U is an excellent wechat bot framework that supports both Node.js and Browser, with rich features and an active community of experienced contributors. PuppetWechat4U will let wechaty users have the second way to use wechaty, not only by Browser Hook but also by Server API Call.
```

**Review by proudofsimin — APPROVED** [review]
add one sentence before the code snippet in the Usage: Run wechaty-puppet-wechat4u.

**Review by huan — APPROVED** [review]
LGTM.

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Pull request merged**